vulnerabilities/importers/apache_kafka.py

```diff
@@ -7,120 +7,179 @@
+            cve_id = cve_section_beginning.get("id")
+
             cve_description_paragraph = cve_section_beginning.find_next_sibling("p")
+
+            description = str(cve_description_paragraph.get_text())
+            description = " ".join(description.split())
+
             cve_data_table = cve_section_beginning.find_next_sibling("table")
             cve_data_table_rows = cve_data_table.find_all("tr")
             affected_versions_row = cve_data_table_rows[0]
             fixed_versions_row = cve_data_table_rows[1]
-            affected_version_ranges = to_version_ranges(
-                affected_versions_row.find_all("td")[1].text
-            )
-            fixed_version_ranges = to_version_ranges(fixed_versions_row.find_all("td")[1].text)
-
-            fixed_packages = [
-                PackageURL(type="apache", name="kafka", version=version)
-                for version in self.version_api.get("apache/kafka").valid_versions
-                if any(
-                    [
-                        MavenVersion(version) in version_range
-                        for version_range in fixed_version_ranges
-                    ]
-                )
-            ]
-
-            affected_packages = [
-                PackageURL(type="apache", name="kafka", version=version)
-                for version in self.version_api.get("apache/kafka").valid_versions
-                if any(
-                    [
-                        MavenVersion(version) in version_range
-                        for version_range in affected_version_ranges
-                    ]
-                )
-            ]
-
-            advisories.append(
-                AdvisoryData(
-                    vulnerability_id=cve_id,
-                    summary=cve_description_paragraph.text,
-                    affected_packages=nearest_patched_package(affected_packages, fixed_packages),
-                    references=[
-                        Reference(url=ASF_PAGE_URL),
-                        Reference(
-                            url=f"https://cve.mitre.org/cgi-bin/cvename.cgi?name={cve_id}",
-                            reference_id=cve_id,
-                        ),
-                    ],
+
+            # Remove leading white space after initial comma
+            affected_versions = affected_versions_row.find_all("td")[1].text
+
+            affected_versions_clean = [v.strip() for v in affected_versions.split(",")]
+            affected_versions_clean = [v for v in affected_versions if v]
+
+            fixed_versions = fixed_versions_row.find_all("td")[1].text
+
+            fixed_versions_clean = [v.strip() for v in fixed_versions.split(",")]
+            fixed_versions_clean = [v for v in fixed_versions if v]
+
+            # This throws a KeyError if the opening h2 tag `id` data changes or is not in the
+            # hard-coded affected_version_range_mapping dictionary.
+            cve_version_mapping = affected_version_range_mapping[cve_id]
```

Comment:
Thanks for the update. I would prefer not having a `_mapping` suffix or similar type suffixes when this is not absolutely needed, but this is cosmetic we can fix this later!